server/src/main/java/org/elasticsearch/action/admin/cluster/settings/ClusterUpdateSettingsResponse.java

```diff
@@ -68,4 +73,31 @@ public void writeTo(StreamOutput out) throws IOException {
         Settings.writeSettingsToStream(persistentSettings, out);
         writeAcknowledged(out);
     }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        builder.startObject();
+        addAcknowledgedField(builder);
+        builder.startObject("persistent");
+        persistentSettings.toXContent(builder, params);
+        builder.endObject();
+        builder.startObject("transient");
```

Comment:
could you make the parse field below constants and reuse them here?

Reply:
Is there a place where I could do this? I mean is a shared consts class which can then be reused by multiple REST classes?